plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/TestingClickHouseServer.java

```diff
@@ -29,14 +29,14 @@
 public class TestingClickHouseServer
         implements Closeable
 {
-    private static final DockerImageName CLICKHOUSE_IMAGE = DockerImageName.parse("yandex/clickhouse-server");
-    public static final DockerImageName CLICKHOUSE_LATEST_IMAGE = CLICKHOUSE_IMAGE.withTag("21.11.10.1");
-    public static final DockerImageName CLICKHOUSE_DEFAULT_IMAGE = CLICKHOUSE_IMAGE.withTag("21.8.14.5"); // EOL is 31 Aug 2022
+    private static final DockerImageName CLICKHOUSE_IMAGE = DockerImageName.parse("clickhouse/clickhouse-server");
+    public static final DockerImageName CLICKHOUSE_LATEST_IMAGE = CLICKHOUSE_IMAGE.withTag("23.8.4.69");
+    public static final DockerImageName CLICKHOUSE_DEFAULT_IMAGE = CLICKHOUSE_IMAGE.withTag("23.3.8.21"); // EOL is Mar 2024
 
     // Altinity Stable Builds Life-Cycle Table https://docs.altinity.com/altinitystablebuilds/#altinity-stable-builds-life-cycle-table
     private static final DockerImageName ALTINITY_IMAGE = DockerImageName.parse("altinity/clickhouse-server").asCompatibleSubstituteFor("yandex/clickhouse-server");
-    public static final DockerImageName ALTINITY_LATEST_IMAGE = ALTINITY_IMAGE.withTag("21.8.13.1.altinitystable");
-    public static final DockerImageName ALTINITY_DEFAULT_IMAGE = ALTINITY_IMAGE.withTag("20.8.4.11_aes"); // EOL is 02 December 2022
+    public static final DockerImageName ALTINITY_LATEST_IMAGE = ALTINITY_IMAGE.withTag("23.3.13.7.altinitystable");
+    public static final DockerImageName ALTINITY_DEFAULT_IMAGE = ALTINITY_IMAGE.withTag("23.3.13.7.altinitystable"); // EOL is 14 July 2026
```

Comment:
Altinity 21.3 is still supported https://docs.altinity.com/altinitystablebuilds/#altinity-stable-builds-life-cycle-table

Reply:
updated to use 21.3

Reply:
@ebyhr 21.3 does not support delete. this was causing some of the tests to fail. 23.3 is the only long term supported version that also support delete.

delete was introduced in [[22.9] ](https://clickhouse.com/docs/en/whats-new/changelog/2022#-clickhouse-release-229-2022-09-22)

https://docs.altinity.com/altinitystablebuilds/#altinity-stable-builds-life-cycle-table

Reply:
I know. Please verify the failure in Altinity test class.

Reply:
@ebyhr : its not just about the test cases. some of the behavior changes too between 21.3 and 23.3 (eg: IPv4/ipv6 mapping logic). also the tests themselves will end up split  - because 21.3 does not support delete whereas 23.3 does. so we will end up having two sets of tests+code. before i make this change, just confirming this was what you meant.